js/bybit.js

```diff
@@ -4895,6 +4897,96 @@ module.exports = class bybit extends Exchange {
         };
     }
 
+    async borrowMargin (code, amount, symbol = undefined, params = {}) {
```

Comment:
Can we add these related exceptions (currently are generic `ExchangeErrors`) ?

```Javascript
ccxt.base.errors.ExchangeError: bybit {"ret_code":-6017,"ret_msg":"Repayment amount has exceeded the total liability","ext_code":null,"ext_info":null,"result":null}

ccxt.base.errors.ExchangeError: bybit {"ret_code":-6026,"ret_msg":"Amount to borrow cannot be larger than the max. amount to borrow (per transaction)","ext_code":null,"ext_info":null,"result":null}

ccxt.base.errors.ExchangeError: bybit {"ret_code":-6025,"ret_msg":"Amount to borrow cannot be lower than the min. amount to borrow (per transaction)","ext_code":null,"ext_info":null,"result":null}

ccxt.base.errors.ExchangeError: bybit {"ret_code":-1103,"ret_msg":"An unknown parameter was sent.","ext_code":null,"ext_info":null,"result":null}

```

Reply:
I've added these all as InvalidOrder exceptions

Reply:
@Dan-krm nice, thanks